sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/NavigationCapability.java

```diff
@@ -0,0 +1,35 @@
+package com.smartdevicelink.proxy.rpc;
+
+import com.smartdevicelink.proxy.RPCStruct;
+
+import java.util.Hashtable;
+
+/*
+ * Extended capabilities for an onboard navigation system
+ */
+public class NavigationCapability extends RPCStruct{
+	public static final String KEY_LOCATION_ENABLED = "sendLocationEnabled";
+	public static final String KEY_GETWAYPOINTS_ENABLED = "getWayPointsEnabled";
+
+	public NavigationCapability(){}
+
+	public NavigationCapability(Hashtable<String, Object> hash) {
+		super(hash);
+	}
+
+	public Boolean getSendLocationEnabled(){
+		return getBoolean(KEY_LOCATION_ENABLED);
+	}
+
+	public void setSendLocationEnabled(Boolean tf){
```

Comment:
Paramater inputs should have relevant names instead of `tf`. For example, this should be `Boolean sendLocationEnabled`